### ISSUE TITLE
Accuracy issue with ONNX conversion

### ISSUE THREAD

**Issue by sangxia** [NVIDIA/TensorRT#2766]
Description
========
I observed large output difference when converting from ONNX to TRT. The issue occurs when I started using `22.12` of the pytorch container on NGC. Previously I have been using `21.08` and there was no problem.

Environment
=========
GPU: 3090
NGC pytorch container 22.12

Steps to reproduce
=============

The files are included in the zip file below.

```
polygraphy run model_folded.onnx --trt --onnxrt --load-inputs input.json
```
Fails (but passes in the 21.08 container)

```
polygraphy run model_folded.onnx --trt --onnxrt --load-inputs input.json --check-error-stat median
```
Fails

```
polygraphy run model_folded.onnx --trt --onnxrt --load-inputs input.json --trt-outputs mark all --onnx-outputs mark all
```
Fails

```
polygraphy run model_folded.onnx --trt --onnxrt --load-inputs input.json --trt-outputs mark all --onnx-outputs mark all --check-error-stat median
```
Passes

[bug.zip](https://github.com/NVIDIA/TensorRT/files/10980770/bug.zip)

**Comment by zerollzeng** [comment]
I can reproduce the issue, will check it later

**Comment by zerollzeng** [comment]
Filed internal bug 4034855 for this.

**Comment by sangxia** [comment]
I've been trying to rewrite the network using different but equivalent layers. After many iterations, I was able to get `polygraphy` to isolate to a much smaller graph with only a GEMM layer followed by a reshape layer. I see in TRT release notes that there is an issue about reshape following GEMM / conv and that it was fixed in 8.5.3. I tried 8.5.3 (with pytorch 23.02 container) but it still has problem. Could you verify this? It would be good to update the doc if that is the case.

In the end, I was able to get around the bug by rewriting the graph so that (1) I use Conv instead of GEMM for linear layers (2) reorder the operations so reshape doesn't follow Conv. It works now with the pytorch 22.12 container.

**Comment by zerollzeng** [comment]
It will be fixed in 8.6 GA. Thanks!

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
